packages/reports/src/Components/account-statistics.tsx

```diff
@@ -3,7 +3,7 @@ import { MobileWrapper, Money, Text } from '@deriv/components';
 import { localize } from '@deriv/translations';
 import { connect } from 'Stores/connect';
 
-type TAccountStatistics = {
+export type TAccountStatistics = {
```

Comment:
@sergei-deriv We shouldn’t export this type as it’s scoped to the `AccountStatistics` component, We can use `React.ComponentProps<typeof AccountStatistics>` to get the props type if needed. But unfortunately `AccountStatistics` component uses the `connect` method internally which messes up the TS so there is no other way :(